src/o365/spo/commands/theme/theme-set.ts

```diff
@@ -0,0 +1,190 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+import * as fs from 'fs';
+import * as path from 'path';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+  filePath: string;
+  isInverted: boolean;
```

Comment:
I think we discussed about this, isInverted needs to be sent in XML payload for UpdateTenantTheme api, if we make isInverted as an optional then we need to make additional call to get the previous state.

Its required in `Add-SPOTheme` PoSh as well - [SharePoint site theming: PowerShell cmdlets](
https://github.com/SharePoint/sp-dev-docs/blob/master/docs/declarative-customization/site-theming/sharepoint-site-theming-powershell.md)

Reply:
But if we make it required, given that we have one command for add and update, it would mean that you always need to set it when updating the theme. If it's optional in the update API, then it should be also optional in the command with a default value.

Reply:
@rjesh-git gotcha, my bad. You're right. Let's keep what you proposed.